public/js/test/mochitest/browser_dbg-scopes.js

```diff
@@ -0,0 +1,27 @@
+/* Any copyright is dedicated to the Public Domain.
+ * http://creativecommons.org/publicdomain/zero/1.0/ */
+
+function toggleNode(dbg, index) {
+  clickElement(dbg, "scopeNode", index);
+}
+
+function getNode(dbg, index) {
+  return findElement(dbg, "scopeNode", index).innerText;
+}
+
+add_task(function* () {
+  const dbg = yield initDebugger("doc-script-switching.html");
+
+  toggleScopes(dbg);
+
+  invokeInTab("firstCall");
+  yield waitForPaused(dbg);
+
+  toggleNode(dbg, 1);
+  toggleNode(dbg, 2);
+
+  yield waitForDispatch(dbg, "LOAD_OBJECT_PROPERTIES");
```

Comment:
One challenge is knowing whether toggling a node will trigger an object load. For the time being this is something the tester has to add, which I think is fine. At some point it might be nice to call `loadObjectProperties` all the time and return properties when they exist in the store. This way the tests could be a little more certain.